0-ad-history-mod-package/simulation/templates/structures/han/corral.xml

```diff
@@ -10,6 +10,9 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Xù Lán</SpecificName>
+    <History>
+       The Land in ancient china was well suited for farming crops, and so that was what most farmers focused their efforts on. However, still most farmers still kept animals. Smaller animals such as pigs, chickens, and goats were kept around the farm for eggs, milk, and meat. Larger animals took more resources to raise but a draft cow was invaluable when plowing.
```

Comment:
```suggestion
       The land in ancient China was well-suited for farming crops, and so that was what most farmers focused their efforts on. However, most farmers still kept animals. Smaller animals such as pigs, chickens, and goats were kept around the farm for eggs, milk, and meat. Larger animals took more resources to raise but a draft cow was invaluable when plowing.
```